core/hello-nft/README.md

```diff
@@ -94,7 +97,11 @@ resim call-method <COMPONENT_ADDRESS> available_ticket_ids
 ```
 resim call-method <COMPONENT_ADDRESS> buy_ticket "100,resource_sim1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzqu57yag"
 ```
-6. Check out our balance
+6. Call the `buy_ticket_by_id` method
+```
+resim call-method <COMPONENT_ADDRESS> buy_ticket_by_id ticket_44 "100,resource_sim1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzqu57yag"
```

Comment:
The new address of XRD is `resource_sim1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz8qety`